packages/webview_flutter/webview_flutter_wkwebview/example/lib/main.dart

```diff
@@ -189,6 +215,14 @@ class _SampleMenu extends StatelessWidget {
               value: _MenuOptions.navigationDelegate,
               child: Text('Navigation Delegate example'),
             ),
+            const PopupMenuItem<_MenuOptions>(
+              value: _MenuOptions.loadHtmlString,
+              child: Text('Load HTML string'),
+            ),
+            const PopupMenuItem<_MenuOptions>(
+              value: _MenuOptions.loadLocalFile,
+              child: Text('Load local file'),
+            ),
```

Comment:
Should we also add integration tests for these in addition to unit tests?

Reply:
That would seem a bit overkill to me to be honest. All calls are covered by unit tests, the only thing the integration tests would add is making sure the WKWebView control indeed loads the page. Since the WKWebView is Apple's control we could safely assume it is working as documented in my opinion.